multigeiger/multigeiger.ino

```diff
@@ -465,10 +465,10 @@ void loop()
         digitalWrite (PIN_SPEAKER_OUTPUT_N, LOW);
         delayMicroseconds(500);
       }
-    }
-
-    if(ledTick & !speakerTick) {
-      delay(4);
+    } else {
```

Comment:
this is now the exact opposite of the if-condition (as it should be), so includes the speaker switch check.